### DISCUSSION TITLE
leap version

### DISCUSSION THREAD

**Discussion by uciharis** [openSUSE/openSUSEway#154] [Ideas]
hello, 
is it possible to make this config come available into leap version please ?

**Comment by FilippoBonazziSUSE** [comment]
Hi, Leap contains older versions of most of the utilities we provide configuration for in openSUSEway. None of our core contributors use Leap, so we do not have a contributor base to maintain a separate openSUSEway version for Leap with its older or missing software.

We are always open to PRs, so feel free to contribute in this area if you're interested.

**Reply by uciharis** [reply]
thank u filippo
